From a12b89b3dcf8d8868b68ac67f54199a2438268da Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Sat, 14 Mar 2020 12:49:16 +0800
Subject: [PATCH] ASoC: fsl: fix a fsl_dma_workaround_free_info not found error
 when compiling fsl_spdif driver

fix a fsl_dma_workaround_free_info not found error when compiling
fsl_spdif driver

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 sound/soc/fsl/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/Makefile b/sound/soc/fsl/Makefile
index 0b23c8e74e41..f7c54c0a775a 100644
--- a/sound/soc/fsl/Makefile
+++ b/sound/soc/fsl/Makefile
@@ -22,7 +22,7 @@ snd-soc-fsl-dsp-objs := fsl_dsp.o fsl_dsp_proxy.o fsl_dsp_pool.o \
 snd-soc-fsl-sai-objs := fsl_sai.o
 snd-soc-fsl-ssi-y := fsl_ssi.o
 snd-soc-fsl-ssi-$(CONFIG_DEBUG_FS) += fsl_ssi_dbg.o
-snd-soc-fsl-spdif-objs := fsl_spdif.o
+snd-soc-fsl-spdif-objs := fsl_spdif.o fsl_dma_workaround.o
 snd-soc-fsl-esai-objs := fsl_esai.o fsl_dma_workaround.o
 snd-soc-fsl-utils-objs := fsl_utils.o
 snd-soc-fsl-dma-objs := fsl_dma.o
-- 
2.17.1

